java-frontend/src/test/files/se/DivisionByZeroCheck.java

```diff
@@ -1,8 +1,8 @@
 class A {
   void foo(int r) {
-    int z1 = 0;
-    int z2 = z1;
-    r = 1 / z2; // Noncompliant [[flows=5]] {{Make sure 'z2' can't be zero before doing this division.}} flow@5 {{z2 is divided by zero}}
+    int z1 = 0; // flow@foo {{z1 is assigned zero value}} flow@foo {{...}}
+    int z2 = z1; // flow@foo {{z2 is assigned zero value}}
```

Comment:
"zero value" sounds a bit weird from user perspective here... I would drop the "value" part